### ISSUE TITLE
Option to not show boosts in Local Timeline

### ISSUE THREAD

**Issue by diogovalentte** [glitch-soc/mastodon#1971]
### Pitch

Have an option to not show boosts in Local Timeline as the Home Timeline has.

### Motivation

50% of my Local Timeline is only boosts and I wanna a way to not see them.

**Comment by ClearlyClaire** [comment]
Hi!
Having boosts in the public or local timelines is not the default server-side behavior, so if that is an issue, it may make sense turning that off.

**Comment by sparky005** [comment]
> Hi! Having boosts in the public or local timelines is not the default server-side behavior, so if that is an issue, it may make sense turning that off.

so does that mean that this is a server-side setting that an admin would have to turn off? I don't see any option as a user to turn off boosts in the local timeline

**Comment by ClearlyClaire** [comment]
Yes, that would be a server-side option that an admin would have to turn off (well, that the admin has turned on).

**Comment by sparky005** [comment]
Gotcha, thanks for clarifying! Is that an option specific to glitch-soc or is it in vanilla mastodon as well? It would be really cool if that was a user-configurable option as well instead of purely just admin side.

**Comment by Plastikmensch** [comment]
> Gotcha, thanks for clarifying! Is that an option specific to glitch-soc or is it in vanilla mastodon as well? It would be really cool if that was a user-configurable option as well instead of purely just admin side.

It's a setting specific to glitch-soc.
I think having a "Show boosts" toggle for users if that setting is enabled is a good idea.

**Comment by horaceyoung** [comment]
> > Gotcha, thanks for clarifying! Is that an option specific to glitch-soc or is it in vanilla mastodon as well? It would be really cool if that was a user-configurable option as well instead of purely just admin side.
> 
> It's a setting specific to glitch-soc. I think having a "Show boosts" toggle for users if that setting is enabled is a good idea.

Would like to plus one on this. Some users would love to see more content but others don't. It would be really nice if it's togglable from user side.
